Typos correction/IPython Notebooks/typos_functions.ipynb

```diff
@@ -0,0 +1,207 @@
+{
```

Comment:
We usually avoid spaces and uppercase in directory and file names. so `typo-correction` and `ipython-notebooks` would look better (or with underscore, I'm not sure if there is any convention for this, maybe underscore is preferred in Python world, cc @vmarkovtsev).